mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -781,7 +780,7 @@ func TestAccComposerEnvironment_withNodeConfig(t *testing.T) {
 		CheckDestroy:             testAccComposerEnvironmentDestroyProducer(t),
 		Steps: []resource.TestStep{
 			{
-				Config: testAccComposerEnvironment_nodeCfg(envName, network, subnetwork, serviceAccount),
+				Config: testAccComposer1Environment_nodeCfg_withTags(envName, network, subnetwork, serviceAccount),
```

Comment:
`testAccComposerEnvironment_nodeCfg` seems to be unused after this change & could be deleted.